swarm/api/routes/runs_control.py

```
@@ -68,6 +70,12 @@ async def pause_run(
         409: Run is not in a pausable state.
         412: ETag mismatch.
     """
+    try:
+        if run_id is not None:
+            validate_path_component(run_id, "run_id")
+    except ValueError as e:
+        raise HTTPException(status_code=400, detail=str(e))
```

Ruff (0.15.5)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)